WS2012R2/lisa/setupscripts/Kdump.ps1

```diff
@@ -257,35 +254,24 @@ else {
 # Give the host a few seconds to record the event
 #
 Write-Output "Waiting seconds to record the event..."
-
-$timeout = 240
-while ( $timeout -gt 0)
-{
-    if ((Get-VMIntegrationService -VMName $vmName -ComputerName $hvServer | ?{$_.name -eq "Heartbeat"}).PrimaryStatusDescription -eq "OK") {
-
-        Write-Output "Info: VM Heartbeat is OK"
-        break
-    }
-    Start-Sleep -S 6
-    $timeout = $timeout -6
-    if ($timeout -eq 0)
-    {
-        Write-Output "Error : Lost Communication to VM" | Tee-Object -Append -file $summaryLog
-        Stop-VM -Name $vmName -ComputerName $hvServer -Force
-        return $false
-    }
+Start-Sleep 10
+
+# Workaround a won't be fixed bug on WS2016 - RHEL6:
+# https://bugzilla.redhat.com/show_bug.cgi?id=1383037
+if ( (-not $RHEL7_Above) -and ($BuildNumber -eq "14393") ){
+    Start-Sleep 120 # Make sure dump completed
+    Stop-VM -vmName $vmName -ComputerName $hvServer -TurnOff
```

Comment:
might I suggest a -Force param here as well?
If we VM is in a blocked state after 120 seconds, the TurnOff command will fail.